packages/integration-sdk-core/src/types/jobState.ts

```diff
@@ -70,6 +70,13 @@ export interface JobState {
    */
   getEntity: (_key: string) => Promise<Entity>;
 
+  /**
+   * Finds an entity by `_key` and returns `null` if the entity does not exist.
+   * This function will also throw an error if multiple entities are found with
+   * the same `_key` value.
```

Comment:
This shouldn't be possible, right, since the JobState should be protecting from writing duplicate keys?

Reply:
Looking at this again, the reason I wondered about having this in the comment is, should users wrap `findEntity` in a `try/catch`, since the docs say it could throw an error. We introduced the function to keep from having to wrap with a `try/catch`. Perhaps the function can be sure to prevent throwing, always returning `null` if the underlying object store throws an error of not found?